documentation/modules/security/proc-renewing-your-own-ca-certificates.adoc

```diff
@@ -5,7 +5,8 @@
 [id='renewing-your-own-ca-certificates-{context}']
 = Renewing your own CA certificates
 
-This procedure describes how to renew CA certificates and keys you installed yourself, instead of using the certificates generated by the Cluster Operator.
+This procedure describes how to renew CA certificates you installed yourself, instead of using the certificates generated by the Cluster Operator.
+Follow this procedure when you are not changing the private keys used to sign the CA certificates, but your new CA certificates are signed with same keys.
```

Comment:
The CA does not have to be always self-signed. It can be just an intermediate CA. So, does this apply when:
* the private key which signed the CA is changed?

or

* When the CA private key is changed?

I assume the second is valid for sure, but never tried the first. Either way should probably make it more clear.

Reply:
Ok this sentence is worded in the wrong way. The CA key is used to sign the server certificates we generate, while I was wrongly mentioning key used to sign the CA which is absolutely a different thing.